samples/snippets/csharp/VS_Snippets_CLR/conceptual.formatting.overview/cs/iformattable.cs

```diff
@@ -69,11 +66,11 @@ public class Example
 {
    public static void Main()
    {
-      Temperature temp1 = new Temperature(22m);
-      Console.WriteLine(Convert.ToString(temp1, new CultureInfo("ja-JP")));
-      Console.WriteLine("Temperature: {0:K}", temp1);
-      Console.WriteLine("Temperature: {0:F}", temp1);
-      Console.WriteLine(String.Format(new CultureInfo("fr-FR"), "Temperature: {0:F}", temp1));
+      Temperature temp = new Temperature(22m);
```

Comment:
Could you also make parallel changes to the `temp` variable name in the Visual Basic example?